### ISSUE TITLE
train a custom model for new obstacles

### ISSUE THREAD

**Issue by belgiumkansas** [CU-Robosub/cusub_perception#16]

**Comment by DrYerzinia** [comment]
This seems like an ongoing task.  We should maybe be more specific and break this into more tasks so we can actually close issues instead of having this open forever.

**Comment by flamma7** [comment]
Yes I agree, probably what we want to change this to is training a model that distinguishes dice5 & dice6, roulette table, path marker & we'll have a mechanical design us a simple torpedo board with 2 openings and a vampire picture & train a model on that. But the labeling and processing of that data are likely separate issues for each task
